pkgs/development/libraries/libpfm/default.nix

```diff
@@ -1,24 +1,34 @@
-{ lib, stdenv, fetchurl
+{ lib
+, stdenv
+, fetchurl
 , enableShared ? !stdenv.hostPlatform.isStatic
+, libgnurx
 }:
 
-stdenv.mkDerivation (rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "4.13.0";
   pname = "libpfm";
 
   src = fetchurl {
-    url = "mirror://sourceforge/perfmon2/libpfm4/${pname}-${version}.tar.gz";
+    url = "mirror://sourceforge/perfmon2/libpfm4/libpfm-${finalAttrs.version}.tar.gz";
     sha256 = "sha256-0YuXdkx1VSjBBR03bjNUXQ62DG6/hWgENoE/pbBMw9E=";
   };
 
+  # Don't install libpfm.so on windows as it doesn't exist
+  # This target is created only if `ifeq ($(SYS),Linux)` passes
+  patches = [ ./fix-windows.patch ];
+
   makeFlags = [
     "PREFIX=${placeholder "out"}"
     "LDCONFIG=true"
     "ARCH=${stdenv.hostPlatform.uname.processor}"
     "SYS=${stdenv.hostPlatform.uname.system}"
-  ];
+  ] ++ lib.optional stdenv.hostPlatform.isWindows "SYS=WINDOWS"; # It must be capitalized
```

Comment:
Is the `uname  -s` capitalized on windows, is upstream wrong

Reply:
`pkgsCross.mingw32.stdenv.hostPlatform.uname.system` gives `"Windows"`, but upstream expects `"WINDOWS"`, see https://github.com/wcohen/libpfm4/blob/70b5b4c82912471b43c7ddf0d1e450c4e0ef477e/config.mk#L251-L253 (mirror)

Reply:
It seems like we're not hitting [this](https://github.com/wcohen/libpfm4/blob/70b5b4c82912471b43c7ddf0d1e450c4e0ef477e/config.mk#L85-L87) for some reason

Reply:
https://github.com/git/git/blob/d814540bb75bbd2257f9a6bf59661a84fe8cf3cf/config.mak.uname#L414

has 
Windows
MINGW

so upstream is wrong, it should be matching `Windows`.

Reply:
Can you substituteInPlace WINDOWS to Windows and add a comment that upstream is wrong.

Reply:
Sure